.buildkite/scripts/validate-tag.sh

```diff
@@ -7,17 +7,26 @@ transformTagAndValidate() {
     local version=$1
     version=${version/#v/}
 
-    if [[ $version =~ ^[0-9]+.[0-9]+.[0-9]+(-[A-Za-z0-9_]+)?$ ]]; then
+    if [[ $version =~ ^[0-9]+.[0-9]+.[0-9]+(-[A-Za-z0-9_]+)?$ || $version == "latest" ]]; then
```

Comment:
As it is now in Jenkins, I think `latest` would not be valid.
https://github.com/elastic/package-registry/blob/main/.ci/release-package-registry-distribution.groovy#L95

Usually, this job is not meant to be run with DOCKER_TAG=latest, so I would not allow it here.

Reply:
I thought it was forgotten from "defaultValue: 'latest'"  https://github.com/elastic/package-registry/blob/448c417ffce864a6c24ad09d0882cb7f97e788e7/.ci/release-package-registry-distribution.groovy#L25 ok, I will remove it then